core/mutator/mutator.go

```diff
@@ -67,7 +67,8 @@ type MutationStorage interface {
 	// ReadAll reads all mutations starting from the given sequence number.
 	// Note that startSequence is not included in the result. ReadAll also
 	// returns the maximum sequence number read.
-	ReadAll(ctx context.Context, mapID int64, startSequence uint64) (uint64, []*pb.EntryUpdate, error)
+	// ReadAll will not return more than count entries.
+	ReadAll(ctx context.Context, mapID int64, startSequence uint64, count int) (uint64, []*pb.EntryUpdate, error)
```

Comment:
Here `count` is `int` but in `ReadRange` it is `int32`.

Reply:
I'll make them both int32

Reply:
ok